.github/workflows/editor-only.yml

```diff
@@ -3,12 +3,16 @@ name: CI - Editor Only
 on:
   push:
     branches:
-      - 'DEFEDIT-*'
       - 'editor-dev'
   pull_request:
     branches:
       - 'DEFEDIT-*'
       - 'editor-dev'
+  pull_request_target:
+    branches:
+       - 'DEFEDIT-*'
+       - 'editor-dev'
+    types: [labeled]
```

Comment:
The editor-only workflow was missing a rule for external PR:s made to the `editor-dev` branch, so I added one.